extended-utxo-spec/extended-utxo-specification.tex

```diff
@@ -135,7 +143,7 @@
 \newcommand{\outputref}{\mi{outputRef}}
 \newcommand{\txin}{\mi{in}}
 \newcommand{\id}{\mi{id}}
-\newcommand{\lookupTx}[2]{#1\langle#2\rangle}
+\newcommand{\lookupTx}[2]{\msf{lookupTx(#1, #2)}}
```

Comment:
Better to just make it a function name like all the others and drop the arguments.

Reply:
Done.